impeller/renderer/backend/vulkan/render_pass_builder_vk.h

```diff
@@ -54,10 +54,19 @@ class RenderPassBuilderVK {
   // Visible for testing.
   const std::optional<vk::AttachmentDescription>& GetDepthStencil() const;
 
+  // Visible for testing.
+  std::optional<vk::AttachmentDescription> GetColor0() const;
+
+  // Visible for testing.
+  std::optional<vk::AttachmentDescription> GetColor0Resolve() const;
+
  private:
+  std::optional<vk::AttachmentDescription> color0_;
+  std::optional<vk::AttachmentDescription> color0_resolve_;
+  std::optional<vk::AttachmentDescription> depth_stencil_;
+
   std::map<size_t, vk::AttachmentDescription> colors_;
```

Comment:
We should add a comment here that colors_[0] is really at color0_.

Reply:
Done